.github/CODEOWNERS

```diff
@@ -376,6 +376,8 @@ packages/react-components/react-color-picker-preview/library @microsoft/cxe-prg
 packages/react-components/react-color-picker-preview/stories @microsoft/cxe-prg
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32966&commitId=c7eb56e4eea8dd59f10d34be62853e84607a3b3f&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|31| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|3| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|4919| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|984| Changed|

</details>


  <details><summary><b>react-charts-preview-DonutChart</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charts-preview-DonutChart.Basic Dark Mode.chromium.png|0| Added|
|react-charts-preview-DonutChart.Basic RTL.chromium.png|0| Added|
|react-charts-preview-DonutChart.Basic.chromium.png|0| Added|
|react-charts-preview-DonutChart.Dynamic Dark Mode.chromium.png|0| Added|
|react-charts-preview-DonutChart.Dynamic RTL.chromium.png|0| Added|
|react-charts-preview-DonutChart.Dynamic.chromium.png|0| Added|

</details>


  <details><summary><b>react-charts-preview-HorizontalBarChart</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charts-preview-HorizontalBarChart.Basic Dark Mode.chromium.png|0| Added|
|react-charts-preview-HorizontalBarChart.Basic RTL.chromium.png|0| Added|
|react-charts-preview-HorizontalBarChart.Basic.chromium.png|0| Added|
|react-charts-preview-HorizontalBarChart.Variant Dark Mode.chromium.png|0| Added|
|react-charts-preview-HorizontalBarChart.Variant RTL.chromium.png|0| Added|
|react-charts-preview-HorizontalBarChart.Variant.chromium.png|0| Added|
|react-charts-preview-HorizontalBarChart.With Benchmark Dark Mode.chromium.png|0| Added|
|react-charts-preview-HorizontalBarChart.With Benchmark RTL.chromium.png|0| Added|
|react-charts-preview-HorizontalBarChart.With_Benchmark.chromium.png|0| Added|

</details>


  <details><summary><b>react-charts-preview-Legend</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charts-preview-Legend.Basic Dark Mode.chromium.png|0| Added|
|react-charts-preview-Legend.Basic RTL.chromium.png|0| Added|
|react-charts-preview-Legend.Basic.chromium.png|0| Added|
|react-charts-preview-Legend.Overflow Dark Mode.chromium.png|0| Added|
|react-charts-preview-Legend.Overflow RTL.chromium.png|0| Added|
|react-charts-preview-Legend.Overflow.chromium.png|0| Added|

</details>


  <details><summary><b>react-charts-preview-LineChart</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charts-preview-LineChart.Basic Dark Mode.chromium.png|0| Added|
|react-charts-preview-LineChart.Basic RTL.chromium.png|0| Added|
|react-charts-preview-LineChart.Basic.chromium.png|0| Added|
|react-charts-preview-LineChart.Events Dark Mode.chromium.png|0| Added|
|react-charts-preview-LineChart.Events RTL.chromium.png|0| Added|
|react-charts-preview-LineChart.Events.chromium.png|0| Added|
|react-charts-preview-LineChart.Gaps Dark Mode.chromium.png|0| Added|
|react-charts-preview-LineChart.Gaps RTL.chromium.png|0| Added|
|react-charts-preview-LineChart.Gaps.chromium.png|0| Added|
|react-charts-preview-LineChart.Multiple Dark Mode.chromium.png|0| Added|
|react-charts-preview-LineChart.Multiple RTL.chromium.png|0| Added|
|react-charts-preview-LineChart.Multiple.chromium.png|0| Added|

</details>


  <details><summary><b>react-charts-preview-SparkLineChart</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charts-preview-SparkLineChart.Basic Dark Mode.chromium.png|0| Added|
|react-charts-preview-SparkLineChart.Basic RTL.chromium.png|0| Added|
|react-charts-preview-SparkLineChart.Basic.chromium.png|0| Added|

</details>


  <details><summary><b>react-charts-preview-VerticalBarChart</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charts-preview-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|0| Added|
|react-charts-preview-VerticalBarChart.Basic Secondary Y Axis Dark Mode.chromium.png|0| Added|
|react-charts-preview-VerticalBarChart.Basic Secondary Y Axis RTL.chromium.png|0| Added|
|react-charts-preview-VerticalBarChart.Date Axis Vbc Dark Mode.chromium.png|0| Added|
|react-charts-preview-VerticalBarChart.Date Axis Vbc RTL.chromium.png|0| Added|
|react-charts-preview-VerticalBarChart.Date Axis- VBC.chromium.png|0| Added|
|react-charts-preview-VerticalBarChart.Dynamic - Wrap Labels.chromium.png|0| Added|
|react-charts-preview-VerticalBarChart.Dynamic Wrap Labels Dark Mode.chromium.png|0| Added|
|react-charts-preview-VerticalBarChart.Dynamic Wrap Labels RTL.chromium.png|0| Added|
|react-charts-preview-VerticalBarChart.Rotated Label Hide Legends Dark Mode.chromium.png|0| Added|
|react-charts-preview-VerticalBarChart.Rotated Label Hide Legends RTL.chromium.png|0| Added|
|react-charts-preview-VerticalBarChart.Rotated Label- Hide Legends.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
New screenshots for chart controls.